### PR TITLE
refactoring scale out doc: using an example to explain it now

### PR DESCRIPTION
**Pull request by xiujiao**
refactor scale out document, using an successful example to explain it now

### CODE REVIEW

File: docs/scale-out.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
+# Scale Out
+
+Scaling out allows adding new service clusters to the existing deployment and adding new node to the management cluster. When new service clusters are added, the new requests for user databases will be distributed among the existing and new service clusters. On one hand, scaling out allows user databases span among more servers; on the other hand, it enables RDPG to flexibly increase user database capacity.
```
Comment (wayneeseguin) [inline docs/scale-out.md]: "adding new" => "adding a new"

File: docs/scale-out.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+yaml
+    resource_pools:
+     - cloud_properties:
+       name: random
+     name: rdpg
+     network: rdpg
+     size: 9
+```
+
+### Testing
```
Comment (wayneeseguin) [inline docs/scale-out.md]: Change "Testing" to "Cheking Cluster Status"

File: docs/scale-out.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+yaml
+    resource_pools:
+     - cloud_properties:
+       name: random
+     name: rdpg
+     network: rdpg
+     size: 10
+```
+
+### Testing
```
Comment (wayneeseguin) [inline docs/scale-out.md]: Same comment.

**Comment by wayneeseguin** [comment]
Overall looks quite good, make those small changes and we'll merge it!

**Comment by xiujiao** [comment]
Thank you , I made changes. Feel good there are less comments from you after writing several docs!

**Pull request merged**